packages/global-cli/README.md

```diff
@@ -1,126 +1,59 @@
+```
 
-### Running the local CLI
+## Future work
 
-Now that the packages are installed in sinopia, you can install the new `react-native-cli` package globally and when you use `react-native init`, it will install the new `react-native` package as well:
+We understand this is counter-intuitive to have two packages for interacting with React Native and it makes the first experience with the framework a bit confusing.
 
-    $ npm uninstall -g react-native-cli
-    $ npm install -g react-native-cli
-    $ react-native init AwesomeApp
-
-## Testing changes
-
-Most of the CLI code is covered by jest tests, which you can run with:
-
-    $ npm test
-
-Project generation is also covered by e2e tests, which you can run with:
-
-    $ ./scripts/e2e-test.sh
-
-These tests actually create a very similar setup to what is described above (using sinopia) and they also run iOS-specific tests, so you will need to run this on OSX and have [xctool] installed.
-
-Both of these types of tests also run on Travis both continuously and on pull requests.
-
-[sinopia]: https://www.npmjs.com/package/sinopia
-[xctool]: https://github.com/facebook/xctool
-
-## Clean up
-
-To unset the npm registry, do:
-
-    $ npm set registry https://registry.npmjs.org/
-    # Check that it worked:
-    $ npm config list
-
-## Troubleshooting
-
-##### Sinopia crashes with "Module version mismatch"
-
-This usually happens when you install a package using one version of Node and then change to a different version. This can happen when you update Node, or switch to a different version with nvm. Do:
-
-    $ npm uninstall -g sinopia
-    $ npm install -g sinopia
-
-After upgrading to Node 4 you might also need to reinstall npm. What worked for me was:
-
-    $ npm uninstall -g npm
-    $ nvm install npm
-
- See the [nvm guide](https://github.com/creationix/nvm#usage) for more info.
-
-### Alternative workflow
-
-If you don't want to install Sinopia you could still test changes done on the cli by creating a sample project and installing your checkout of `react-native` on that project instead of downloading it from npm. The simplest way to do this is by:
-
-    $ npm init AwesomeProject
-    $ cd AwesomeProject
-    $ npm install $REACT_NATIVE_GITHUB
-
-Note that `REACT_NATIVE_GITHUB` should point to the directory where you have a checkout.
-
-Also, if the changes you're making get triggered when running `react-native init AwesomeProject` you will want to tweak the global installed `react-native-cli` library to install the local checkout instead of downloading the module from npm. To do so just change this [line](https://github.com/facebook/react-native/blob/master/react-native-cli/index.js#L191) and refer the local checkout instead.
+That's why, as a part of [Lean Core](https://github.com/facebook/react-native/issues/23313) initiative, there's an ongoing effort to remove this module so we can use just `react-native` as the only package necessary to install and run React Native commands.
```

Comment:
We could reference the issue where the discussion happens

Reply:
linked